docs/blog/posts/2021-10-06-sharedTest.md

```diff
@@ -142,21 +144,22 @@ There are some Google's projects have used `sharedTest` pattern to sharing test
 
 - [accompanist: [All] Share tests to run on Robolectric & Emulators by chrisbanes][10]
 
+<!-- markdownlint-disable MD013 -->
```

Comment:
Because of the link titles used below, some lines are too long. I'll also remove them when they are redundant in a separate PR.

https://github.com/DavidAnson/markdownlint/blob/v0.36.0/doc/md013.md